pkg/document/time/actor_id.go

```diff
@@ -65,6 +65,11 @@ func ActorIDFromHex(str string) (*ActorID, error) {
 	if err != nil {
 		return nil, fmt.Errorf("%s: %w", str, ErrInvalidHexString)
 	}
+
+	if len(decoded) != actorIDSize {
+		return nil, fmt.Errorf("%s: %w decode size: %d", str, ErrInvalidHexString, len(decoded))
```

Comment:
When wrapping an error using `fmt.Errorf`, it is best practice to put the sentinel error behind.

```suggestion
		return nil, fmt.Errorf("decoded length %d: %w", len(decoded), ErrInvalidHexString)
```

Would you like to refer to the article below?
https://blog.golang.org/go1.13-errors

Reply:
Thank you for your comment, It would be better I will edit it soon :)